blueprints/data-solutions/dataproc-lifecycle-via-composer/README.md

```diff
@@ -0,0 +1,115 @@
+# Dataproc lifecycle management orchestrated by Composer
+
+Writing DAGs isn’t practical when having multiple DAGs that run similar Dataproc Jobs, and want to share clusters efficiently, with just some parameters changing between them. Here makes sense to dynamically generate DAGs.
+
+Using this project you can deploy multiple Composer/Airflow DAGs from a single configuration, it means you will create configuration files for DAGs to be automatically generated during deployment _(you can also use it to generate and upload DAGs manually)_.
+
+Each DAG will execute a single Dataproc cluster Job referenced in a DAG configuration file, and that Job will be executed in a Dataproc Cluster that can be reused for multiple Jobs/DAGs. 
+
+The Dataproc cluster lifecycle management will be done by the automatically generated Airflow DAGs, to reuse or create clusters accordingly _(cluster proposed configuration includes a scalability policy usage)_. 
+
+This approach aims to use resources efficiently meanwhile minimize provisioning and execution time.
+
+This is the high level diagram:
+![](images/dataproc_lifecycle.png)
+
+
+## Prerequisites
+1. This blueprint will deploy all its resources into the project defined by the `project_id` variable. Please note that we assume this project already exists.
+2. The user deploying the project _(executing terraform plan/apply)_ should have admin permissions in the selected project, or permission to create all the resources in the Terraform **main.tf** script. 
+
+## Project Folder Structure
+ ```bash
+ terraform/
```

Comment:
why not having the terraform files in the root folder of this example?

Reply:
tf files moved